applications/StructuralMechanicsApplication/custom_processes/spr_error_process.h

```diff
@@ -0,0 +1,308 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:		 BSD License
+//					 license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Anna Rehr
+//  Co-author   :    Vicente Mataix Ferrandiz
+//
+
+#if !defined(KRATOS_SPR_ERROR_PROCESS)
```

Comment:
why "SPR"? wouldn't it make sense to call it Zienkiewicz Zhu Error estimation process or something of the sort?

Reply:
The official name is SPR, Super Patch Recovery, look at the papers mentioned in the header

Reply:
ok then, just ensure that the acronym is defined somewhere in the header

Reply:
The definiton of SPR is on the header description

Reply:
ok